Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -84,8 +85,6 @@ namespace Multiplayer
     AZ_CVAR(bool, sv_isDedicated, true, nullptr, AZ::ConsoleFunctorFlags::DontReplicate, "Whether the host command creates an independent or client hosted server");
     AZ_CVAR(bool, sv_isTransient, true, nullptr, AZ::ConsoleFunctorFlags::DontReplicate, "Whether a dedicated server shuts down if all existing connections disconnect.");
     AZ_CVAR(AZ::TimeMs, sv_serverSendRateMs, AZ::TimeMs{ 50 }, nullptr, AZ::ConsoleFunctorFlags::Null, "Minimum number of milliseconds between each network update");
-    AZ_CVAR(AZ::CVarFixedString, sv_defaultPlayerSpawnAsset, "prefabs/player.network.spawnable", nullptr, AZ::ConsoleFunctorFlags::DontReplicate,
```

Comment:
No, MultiplayerSample will. The main point is that this responsibility is moved to game code that uses the Multiplayer Gem. See https://github.com/o3de/o3de-multiplayersample/pull/106

Reply:
My hesitation of not having a default is that it will add an extra step for devs wanting to get started with multiplayer (a process which already requires extra steps over other gems which are plug and play). 🤔

Reply:
Again, see the MPSample pull request. The default is over there now.

Reply:
The MPS Sample, is a perfect example. I'm saying it's _MultiplayerSample/Gem/Code/Source/MultiplayerSampleSystemComponent.h_
As opposed to _o3de/Gems/Multiplayer/Code/..._
It'll require most (all?) devs to do some copy+pasting after enabling the MultiplayerGem.
The Multiplayer Gem already requires some copy+pasting, but I notice ppl at first are confused by that since the other gems are plug and play.
One answer could be adding the Sample to the Multiplayer Template, although I think it would be cooler to have MP gem handle these sort of things automatically.

Reply:
To put it another way, I'm having trouble imagining anyone not wanting to use your sweet spawner code when getting started :P